diff --git a/ext/mysql/config.m4 b/ext/mysql/config.m4
index 5870be0..f18ad80 100644
--- a/ext/mysql/config.m4
+++ b/ext/mysql/config.m4
@@ -105,7 +105,7 @@ Note that the MySQL client library is not bundled anymore!])
   for i in $PHP_LIBDIR $PHP_LIBDIR/mysql; do
     MYSQL_LIB_CHK($i)
   done
-
+MYSQL_LIB_DIR=/usr/lib
   if test -z "$MYSQL_LIB_DIR"; then
     AC_MSG_ERROR([Cannot find lib$MYSQL_LIBNAME under $MYSQL_DIR.
 Note that the MySQL client library is not bundled anymore!])
