--- a/ext/standard/config.m4.orig	2009-11-27 17:41:13.000000000 -0600
+++ b/ext/standard/config.m4	2010-07-23 06:21:42.000000000 -0500
@@ -311,9 +311,9 @@
 dnl Detect library functions needed by php dns_xxx functions
 dnl ext/standard/php_dns.h will collect these in a single define: HAVE_DNS_FUNCS
 dnl
-PHP_CHECK_FUNC(res_nmkquery, resolv, bind, socket)
-PHP_CHECK_FUNC(res_nsend, resolv, bind, socket)
-PHP_CHECK_FUNC(dn_expand, resolv, bind, socket)
+PHP_CHECK_FUNC(res_nmkquery, resolv, socket)
+PHP_CHECK_FUNC(res_nsend, resolv, socket)
+PHP_CHECK_FUNC(dn_expand, resolv, socket)

 dnl OSX has the dns functions in libc but remaps them in resolv.h for bind so linking fails
 case $host_alias in
